src/Symfony/Component/Translation/Dumper/XliffFileDumper.php

```diff
@@ -21,29 +21,20 @@
 class XliffFileDumper extends FileDumper
 {
     /**
-     * @var string
-     */
-    private $defaultLocale;
-
-    /**
      * {@inheritdoc}
      */
-    public function dump(MessageCatalogue $messages, $options = array())
+    protected function formatCatalogue(MessageCatalogue $messages, $domain, array $options = array())
     {
+        $defaultLocale = \Locale::getDefault();
```

Comment:
I would avoid doing that call when the default locale is specified. It will avoid the dependency on Intl (or the stub) when you specify the default locale explicitly). The old code was supporting this